picture.go

```diff
@@ -16,6 +16,7 @@ import (
 	"errors"
 	"image"
 	"io/ioutil"
+	"log"
```

Comment:
I propose to return an error to the caller of this function instead of log errors.

Reply:
In your library design, a lot of functions do not return an error, but an error is possible inside the function. There is an option to use panic, but it is much worse than sending an error to the log.

Everywhere where it is possible to return an error from a function, an error is returned, where the design does not provide for an error, I had to insert a log.